tests/v2/integration/api/auto_runner/test_auto_runner.py

```diff
@@ -5,12 +5,23 @@
 
 import pytest
 from otx.v2.api.core import AutoRunner
+from otx.v2.api.entities.task_type import TaskType
 
 from tests.v2.integration.test_helper import TASK_CONFIGURATION
 
 # NOTE: This test currently only checks the basic pipeline and doesn't do much checking on the result, which will be fixed in the future.
 
 class TestAutoRunnerAPI:
+    @pytest.fixture(autouse=True)
+    def setup(self):
+        self.task_type_dict = {
```

Comment:
again, not sure if i understand why we need this manual mapping. would it be possible to get this directly from the enum?